tests/kernel/update_kernel.pm

```diff
@@ -19,6 +19,8 @@ use utils;
 use version_utils 'is_sle';
 use qam;
 use kernel 'remove_kernel_packages';
+use power_action_utils 'power_action';
+
 
```

Comment:
+ empty line?

Reply:
and ?

Reply:
two(instead of one) empty lines between lines 22 and 25

Reply:
@mimi1vx is sulky :wink: